src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/IssueCreateDialog.kt

```diff
@@ -70,10 +70,12 @@ class IssueCreateDialog(
      * Click 'Configure Fields' to display 'All' to ensure
      * all mandatory fields are displayed in creation dialog.
      *
+     * The view has changed since Jira 8.19 so differentiator is needed
```

Comment:
If you want to increase CI coverage, the Jira versions are set in `.github/workflows/ci.yml`

Reply:
Currently, I don't as 8.19 is not released yet.